src/java.base/share/classes/java/io/BufferedReader.java

```diff
@@ -394,12 +398,74 @@ String readLine(boolean ignoreLF, boolean[] term) throws IOException {
      *
      * @throws     IOException  If an I/O error occurs
      *
+     * @see #readAllLines
      * @see java.nio.file.Files#readAllLines
      */
     public String readLine() throws IOException {
         return readLine(false, null);
     }
 
+    /**
+     * Reads all lines of text. A line is considered to be terminated by any
+     * one of a line feed ('\n'), a carriage return ('\r'), a carriage return
+     * followed immediately by a line feed, or by reaching the end-of-file
+     * (EOF).
+     *
+     * <p> This method works as if invoking it were equivalent to evaluating
+     * the expression:
+     * <blockquote>{@link #lines()}.toList()</blockquote>
+     *
+     * @apiNote
+     * <p> This method is intended for simple cases where it is convenient
```

Comment:
```suggestion
     * This method is intended for simple cases where it is convenient
```
API notes already have a dd/dt block, no need for extra paragraphs.

Reply:
Thanks. Fixed in b5d8266.